modules/tinymce/src/core/main/ts/textpatterns/core/InlinePattern.ts

```diff
@@ -123,7 +123,7 @@ const findPattern = (editor: Editor, block: Node, details: PatternDetails): Opti
     } else {
       // Find any nested patterns, making sure not to process the current pattern again
       const resultsOpt = findPatternsRec(editor, details.remainingPatterns, spot.container, spot.offset, block);
-      const results = resultsOpt.getOr({ matches: [], position: spot });
+      const results = resultsOpt.getOrThunk((): SearchResults => ({ matches: [], position: spot }));
```

Comment:
it seems weird that we have to introduce a constant function just for the types 🤔

Reply:
Depends whether you consider an eager object or eager function to be more expensive 🤔

Reply:
Yeah that's also a good point... it's probably negligible and the performance changes we're talking about isn't going to matter here, so I'll swap it back to the above instead.
```suggestion
      const results: SearchResults = resultsOpt.getOr({ matches: [], position: spot });
```